### PR TITLE
(submit) event preconditions edit

### PR DESCRIPTION
**Pull request by skrikl**
Current document may confuse the reader by guiding how to emit a (submit) event. There are some other options to consider and one to be aware of (type='button').

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Documentation is possibly misleading
Issue Number: N/A

## What is the new behavior?
Documentation changed

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: aio/content/guide/reactive-forms.md [hunk 1]
```diff
@@ -168,7 +168,7 @@ The `onSubmit()` method in the `ProfileEditor` component captures the current va
 
 </code-example>
 
-The `submit` event is emitted by the `form` tag using the native DOM event. You trigger the event by clicking a button with `submit` type. This allows the user to press the **Enter** key to submit the completed form.
+The `submit` event is emitted by the `form` tag using the native DOM event. You trigger the event by clicking an <input> element with a `submit` type, or a <button> element with a type other than 'button'. This also allows the user to press the **Enter** key to submit the completed form.
```
Comment (kapunahelewong) [inline aio/content/guide/reactive-forms.md]: Though this was already there, can you edit the first sentence so it's no longer in the passive voice? And can you put `<input>`, `<button>`, and `button` in backticks so they're in code font? Like this:
```suggestion
The `form` tag uses the native DOM event to emit the `submit` event. You trigger the event by clicking an `<input>` element with a `submit` type, or a `<button>` element with a type other than `button`. This also allows the user to press the **Enter** key to submit the completed form.
```

**Review by kapunahelewong — CHANGES_REQUESTED** [review]
Nice update, @skrikl. Thank you. Just some minimal copy edits. Could you also [amend your commit message](https://gist.github.com/kapunahelewong/567c30f7b353fd1c39bbf6d8343e2dca#how-to-amend-a-commit) to something like:

> docs: edit event preconditions in reactive forms doc

The important part is the format of `docs:....` because without it the test fails. Otherwise, almost there. 🌻

File: aio/content/guide/reactive-forms.md (hunk 1 above)
Comment (skrikl) [reply aio/content/guide/reactive-forms.md]: Done, I hope I did it right :)

**Comment by kapunahelewong** [comment]
Thanks for the additional updates, @skrikl. Requesting @jbogarthyde to review as she may have feedback on style choices, too.

Just as a note for later (that you probably already know), you'll ultimately want everything [squashed](https://gist.github.com/kapunahelewong/567c30f7b353fd1c39bbf6d8343e2dca#how-to-squash) into one commit.

File: aio/content/guide/reactive-forms.md [hunk 2]
```diff
@@ -168,7 +166,7 @@ The `onSubmit()` method in the `ProfileEditor` component captures the current va
 
 </code-example>
 
-The `submit` event is emitted by the `form` tag using the native DOM event. You trigger the event by clicking a button with `submit` type. This allows the user to press the **Enter** key to submit the completed form.
+The `form` tag uses the native DOM event to emit the `submit` event. You trigger the event by clicking an `<input>` element with a `submit` type, or a `<button>` element with a type other than `button`. This also allows the user to press the **Enter** key to submit the completed form.
```
Comment (jbogarthyde) [inline aio/content/guide/reactive-forms.md]: When you say "This allows...", what do you mean by "this"? Is it just the <button> tag?  Please be more specific - perhaps something like "Using a button allows the user..."

**Review by jbogarthyde — CHANGES_REQUESTED** [review]
A good change. One minor point. 

We generally try standardize on asterisks rather than underscores and hyphens to indicate italics and bullets, but it doesn't really make much difference. No need to change them, but no need to change them back 😃

**Comment by AndrewKushnir** [comment]
Closing this PR due to the lack of reply. Please reopen if needed. Thank you.

**Comment by theBolck** [comment]
skrikl:patch-2

**Pull request closed without merging**